modules/virt-viewing-component-logs-cli.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-logs.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-logs-cli_{context}"]
+= Viewing {VirtProductName} logs with the CLI
+
+Configure log verbosity for {VirtProductName} components by editing the `HyperConverged` custom resource (CR). Then, view logs for the component pods by using the `oc` CLI tool.
+
+.Procedure
+
+. To set log verbosity for specific components, open the `HyperConverged` CR in your default text editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Set the log level for one or more components by editing the `spec.configuration.developerConfiguration.logVerbosity` stanza. For example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  configuration:
+    developerConfiguration:
+      logVerbosity:
+        virtLauncher: 2 <1>
+        virtHandler: 3
+        virtController: 4
+        virtAPI: 5
+        virtOperator: 6
+----
+<1> The log verbosity value must be an integer in the range `1–9`, where a higher number indicates a more detailed log. In this example, the `virtLauncher` component logs are exposed if they are greater than or equal to a verbosity level of `2`.
```

Comment:
I just want to point out that, at least in my opinion, we should determine a much clearer definition of each verbosity level. I've also opened [an issue](https://github.com/kubevirt/kubevirt/issues/8164) on that subject.

As I see it, we would end up with something like 4-5 level that are `Info`, `Debug`, `Warnings`, etc.

Maybe that's worth mentioning here? @stu-gott WDYT?

Reply:
+1, I agree that the levels should be more clearly defined. We shouldn't mention features that aren't implemented yet, but we'd be happy to update the docs later.